diff/testdata/gen.sh

```diff
@@ -11,3 +11,4 @@ git diff --no-index /dev/null "empty space.txt" > empty_space.diff
 git diff --no-index golint.{old,new}.go > golint.diff
 git diff --no-index empty.txt /dev/null > newline_and_empty_deleted.diff
 git diff --no-index golint.{old,new}.go >> newline_and_empty_deleted.diff
+git diff --no-index \"日本語\".{old,new}.txt >> \"日本語\".diff
```

Comment:
Can you use `diff -u` so that this script works when core.quotePath is false?

Or specify flag to turn on quotePath if there are such flag (I cannot find it from a quick read...).

Reply:
I found `-c` option https://git-scm.com/docs/git#Documentation/git.txt--cltnamegtltvaluegt and used it. https://github.com/haya14busa/reviewdog/pull/183/commits/8ed450a7eb7ab28a578042bce83c5b573e91e47c